osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -40,6 +40,13 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
 
             double sliderFactor = aimRating > 0 ? aimRatingNoSliders / aimRating : 1;
 
+            int aimDifficultyStrainCount = ((OsuStrainSkill)skills[0]).RelevantDifficultStrains();
+            int speedDifficultyStrainCount = ((OsuStrainSkill)skills[2]).RelevantDifficultStrains();
+
+            // Total number of strains in a map can vary by clockrate, and this needs to be corrected for.
+            aimDifficultyStrainCount = (int)(aimDifficultyStrainCount * clockRate);
```

Comment:
Make `RelevantDifficultStrains` have clock rate as param instead? So that it'll be `((OsuStrainSkill)skills[0]).RelevantDifficultStrains(clockRate);` and `public int RelevantDifficultStrains(double clockRate)`

Reply:
Yup, good idea.